src/main/kotlin/uk/gov/justice/digital/hmpps/hmppsrestrictedpatientsapi/model/response/SubjectAccessRequestContent.kt

```diff
@@ -0,0 +1,28 @@
+package uk.gov.justice.digital.hmpps.hmppsrestrictedpatientsapi.model.response
+
+import io.swagger.v3.oas.annotations.media.Schema
+import java.time.LocalDateTime
+
+data class SubjectAccessRequestContent(
+  @Schema(description = "The content of the subject access request response")
+  val content: RestrictedPatientContent,
+)
+
+data class RestrictedPatientContent(
+  @Schema(description = "The prisoner number (Nomis ID)", example = "A1234AA")
+  val prisonerNumber: String,
+  @Schema(
+    description = "Prison where the offender is supported by POM.  This is from AGENCY_LOCATIONS.AGY_LOC_ID in NOMIS.",
+    example = "LEI",
+  )
+  val supportingPrisonId: String,
```

Comment:
I've opted for only supplying the data we've got for now and haven't done lookups to Prison API.  Will add them later if needed

Reply:
also the supporting prison isn't held by Prison API so we return it - can be different from the discharge prison (which will be held in NOMIS as the last movement from location)